src/ngraph/op/fused/gru_cell.hpp

```diff
@@ -65,55 +65,59 @@ namespace ngraph
             ///
             /// \brief      Constructs GRUCell node.
             ///
-            /// \param[in]  X                 The input tensor with shape: [batch_size, input_size].
-            /// \param[in]  W                 The weight tensor with shape:
-            ///                               [gates_count * hidden_size, input_size].
-            /// \param[in]  R                 The recurrence weight tensor with shape:
-            ///                               [gates_count * hidden_size, hidden_size].
-            /// \param[in]  H_t               The hidden state tensor at current time step with
-            ///                               shape: [batch_size, hidden_size].
-            /// \param[in]  hidden_size       The number of hidden units for recurrent cell.
-            /// \param[in]  activations       The vector of activation functions used inside
-            ///                               recurrent cell.
-            /// \param[in]  activation_alpha  The vector of alpha parameters for activation
-            ///                               functions in order respective to activation list.
-            /// \param[in]  activation_beta   The vector of beta parameters for activation functions
-            ///                               in order respective to activation list.
-            /// \param[in]  clip              The value defining clipping range [-clip, clip] on
-            ///                               input of activation functions.
+            /// \param[in]  X                  The input tensor with shape: [batch_size,
+            ///                                input_size].
+            /// \param[in]  W                  The weight tensor with shape:
+            ///                                [gates_count * hidden_size, input_size].
+            /// \param[in]  R                  The recurrence weight tensor with shape:
+            ///                                [gates_count * hidden_size, hidden_size].
+            /// \param[in]  H_t                The hidden state tensor at current time step with
```

Comment:
I think we should have the same names for the similar inputs of the LSTMCell, LSTMSequence, GRUCell, and RNN Cell operations.  I suggest rename H_t, C_t to initial_hidden_state, initial_cell_state as in LSTMSequence operation. I will fix the specification on our side.

Reply:
Done